driver/node.go

```diff
@@ -447,8 +472,101 @@ func (d *Driver) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandVolume
 	return &csi.NodeExpandVolumeResponse{}, nil
 }
 
-// getDiskSource returns the absolute path of the attached volume for the given
+func (d *Driver) nodePublishVolumeForFileSystem(req *csi.NodePublishVolumeRequest, mountOptions []string, log *logrus.Entry) error {
+	source := req.StagingTargetPath
+	target := req.TargetPath
+
+	mnt := req.VolumeCapability.GetMount()
+	for _, flag := range mnt.MountFlags {
+		mountOptions = append(mountOptions, flag)
+	}
+
+	fsType := "ext4"
+	if mnt.FsType != "" {
+		fsType = mnt.FsType
+	}
+
+	mounted, err := d.mounter.IsMounted(target)
+	if err != nil {
+		return err
+	}
+
+	log.WithFields(logrus.Fields{
```

Comment:
`log.WithFields()` needs to be assigned to `log` again or otherwise the extra fields won't show up in the `log.Info()` statements below.

Reply:
👍